rpc/src/v1/impls/light/parity.rs

```diff
@@ -132,6 +132,11 @@ impl Parity for ParityClient {
 		)
 	}
 
+	fn trezor(&self, message_type: String, device_path: Option<String>, message: Option<String>) -> Result<String, Error> {
```

Comment:
Imho the API is quite unsafe - it allows you to send arbitrary messages to a connected trezor device by any dapp.

We should:
1. Move that API to `parity_set` (protected set of methods) or even `parity_accounts`
2. Restrict the RPC and introduce enum for allowed messages + the messages should be constructed on parity side.

Reply:
It doesn't allow sending arbitrary messages _to the device_. There's plenty of steps to validate the data before it hits the device, ultimately the only thing that is sent to the device is a "pin" which is sent through protobuf and is "supposed" to be safe, however as an extra safety precaution we could make sure that the input string isn't more than 9 characters, this is however already done by the trezor firmware and IMO should be their responsibility to make sure that the input pin is safety-checked.